contents/docs/integrate/_snippets/install-go.mdx

```diff
@@ -15,7 +15,6 @@ func main() {
 		os.Getenv("POSTHOG_API_KEY"),
 		posthog.Config{
 			Endpoint:       "<ph_instance_address>",
-			PersonalApiKey: "your personal API key", // needed for feature flags
```

Comment:
looks like go indeed has a requirement for personal api key 🤔 , so revert here.

I think this used to be the case for all libs, until we updated to make this optional, but hasn't happened in go

Reply:
Yikes. Good catch